flow/test/test_helper.sh

```diff
@@ -59,8 +60,13 @@ fi
 # Only enabled abort on error at the end to allow script to reach make issue
 set -e
 
-if [ ! -z ${MAKE_ISSUE+x} ]; then
-  $__make final_report_issue 2>&1 | tee -a $LOG_FILE
+if [ -n "${MAKE_ISSUE+x}" ]; then
+  $__make final_report_issue 2>&1 | tee -a "$LOG_FILE"
+fi
+
+if [ $ret -eq 0 ] && [ -f "$POWER_TCL_FILE" ]; then
+  $__make simulate power 2>&1 | tee -a "$LOG_FILE"
```

Comment:
Where is this target defined?

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/flow/designs/asap7/mock-array/config.mk#L50